packages/template-retail-react-app/app/hooks/use-einstein.test.js

```diff
@@ -15,28 +14,30 @@ import {
     mockRecommenderDetails
 } from './einstein-mock-data'
 
-jest.mock('cross-fetch', () => {
-    return {
-        __esModule: true,
-        default: jest.fn(() => ({json: jest.fn(), ok: true}))
-    }
-})
-
 const einsteinApi = new EinsteinAPI({
     host: `http://localhost/test-path`,
     einsteinId: 'test-id',
     siteId: 'test-site-id',
     cookieId: 'test-usid'
 })
 
-beforeEach(() => {
-    jest.resetModules()
-    fetch.mockClear()
+const fetchMock = require('jest-fetch-mock')
```

Comment:
Are we installing this package?

Reply:
The lib was already installed and in dev dependancies. But I'm not sure what it was used for before. I think it was used before MSW as added maybe?

Anyway, it doesn't hurt to use a mock fetch lib that was already installed, rather than a one off mock. It might give more flexibility in the future if we have tests that do more than mock a successful call.